server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -2522,12 +2522,15 @@ public void clusterStateProcessed(String source, ClusterState oldState, final Cl
         assertAcked(client.admin().cluster().prepareDeleteSnapshot(repositoryName, snapshotName).get("10s"));
     }
 
-    private boolean waitForIndex(final String index, TimeValue timeout) throws InterruptedException {
-        return awaitBusy(() -> client().admin().indices().prepareExists(index).execute().actionGet().isExists(), timeout.millis(), TimeUnit.MILLISECONDS);
+    private void waitForIndex(final String index, TimeValue timeout) throws Exception {
+        assertBusy(() -> assertTrue(client().admin().indices().prepareExists(index).execute().actionGet().isExists()),
+            timeout.millis(), TimeUnit.MILLISECONDS);
     }
 
-    private boolean waitForRelocationsToStart(final String index, TimeValue timeout) throws InterruptedException {
-        return awaitBusy(() -> client().admin().cluster().prepareHealth(index).execute().actionGet().getRelocatingShards() > 0, timeout.millis(), TimeUnit.MILLISECONDS);
+    private void waitForRelocationsToStart(final String index, TimeValue timeout) throws Exception {
+        assertBusy(() -> assertThat(
```

Comment:
@javanna This assert is failed. I don't know how to make it passed. Could you help me?

Reply:
silly one, but took me a while to figure this out... we were previously not asserting on the return value of `awaitBusy`, hence the wait was not really doing anything, only sleeping for ~300ms without any verifications. I think we can remove this wait from the test completely. It should be green without it.